apps/studio/components/interfaces/Integrations/CronJobs/CronJobCard.tsx

```diff
@@ -68,19 +73,47 @@ export const CronJobCard = ({ job, onEditCronJob, onDeleteCronJob }: CronJobCard
                 checked={job.active}
                 onCheckedChange={() => showToggleConfirmationModal(true)}
               />
-              <Button type="default" icon={<History />}>
+              <Button
+                type="default"
+                icon={<History />}
+                onClick={() => {
+                  sendEvent({
+                    action: TELEMETRY_EVENTS.CRON_JOBS,
+                    value: TELEMETRY_VALUES.CRON_JOBS_VIEW_PREVIOUS_RUNS,
+                    label: 'User clicked view previous runs button',
+                  })
+                }}
+              >
                 <Link href={`/project/${ref}/integrations/cron/jobs/${job.jobname}`}>History</Link>
               </Button>
               <DropdownMenu>
                 <DropdownMenuTrigger asChild>
                   <Button type="default" icon={<MoreVertical />} className="px-1.5" />
                 </DropdownMenuTrigger>
                 <DropdownMenuContent align="end" className="w-36">
-                  <DropdownMenuItem onClick={() => onEditCronJob(job)}>
+                  <DropdownMenuItem
+                    onClick={() => {
+                      sendEvent({
+                        action: TELEMETRY_EVENTS.CRON_JOBS,
+                        value: TELEMETRY_VALUES.CRON_JOB_UPDATE_CLICKED,
+                        label: 'User clicked edit cron job button',
+                      })
+                      onEditCronJob(job)
+                    }}
+                  >
                     Edit cron job
                   </DropdownMenuItem>
                   <DropdownMenuSeparator />
-                  <DropdownMenuItem onClick={() => onDeleteCronJob(job)}>
+                  <DropdownMenuItem
+                    onClick={() => {
+                      sendEvent({
+                        action: TELEMETRY_EVENTS.CRON_JOBS,
+                        value: TELEMETRY_VALUES.CRON_JOB_DELETE_CLICKED,
+                        label: 'User clicked delete cron job button',
+                      })
```

Comment:
This event triggers only when the modal opens and not when they actually delete a cron job – is that okay?
(same with the edit one above)

Reply:
yeah - i think this is okay for now, we'll track both "clicked" and "actually deleted" - which reminds me i need to add the latter haha

Reply:
we need to think through properly though what we want to track and what for - but since we're launching today, we'll just add this in to get _some_ telemetry